cql/fhir-cql-server/src/test/java/com/ibm/fhir/cql/engine/server/retrieve/ServerFHIRRetrieveProviderTest.java

```diff
@@ -50,6 +52,11 @@
     private ServerFHIRRetrieveProvider provider;
     private FHIRResourceHelpers helpers;
 
+    @BeforeClass
+    public void initializeSearchUtil() {
```

Comment:
agree, but we already have that issue due to statics in FHIRUtil.  i decided to stop at just ParametersUtil and CompartmentUtil for this PR.  changing SearchUtil to avoid statics will be a large undertaking and I'm not sure its worth it at this time

Reply:
after much consternation, I've now added a commit that converts most SearchUtil methods from static to instance-level.  this addresses the race condition concern, but forced numerous other changes/decisions and so a new review is needed